### DISCUSSION TITLE
Add DISTINCT in select dropdown

### DISCUSSION THREAD

**Discussion by SanderVanKeer** [adminerevo/adminerevo#27] [Ideas]
In the select dropdown of a table (see screenshot), it would be nice to have the option "distinct" available. For now only "count_distinct" is available.

This would result in a query like this: "SELECT DISTINCT `foo` FROM `bar`.

![image](https://github.com/adminerevo/adminerevo/assets/10955678/a29d2805-a3a5-4015-af6d-8011c5d80ae3)

**Comment by LionelLaffineur** [comment]
Will be part of the upcoming v4.8.2.
Thanks for submitting the idea !
